ansible/ncn_nodes.yml

```
@@ -74,3 +74,8 @@
       vars:
         packages: "{{ common_csm_sles_packages + common_mgmt_ncn_csm_sles_packages }}"
       when: ansible_os_family == "SLE_HPC"
+
+    - role: csm.storage.smartmon
+      vars:
+        packages: "{{ storage_csm_noos_packages }}"
```

It makes more sense to add the packages to the existing package install play. And the name of the packages should be changed to match the pattern of the other names

```suggestion
        packages: "{{ common_csm_sles_packages + common_mgmt_ncn_csm_sles_packages + storage_mgmt_ncn_csm_sles_packages }}"
      when: ansible_os_family == "SLE_HPC"

    - role: csm.storage.smartmon
```